style.css

```diff
@@ -0,0 +1,410 @@
+@media only screen and (min-width: 1080px) {
+  nav {
+    position: sticky;
+    top: 0;
+    background-color: #fff;
+  }
+}
+
+@media (min-width: 1300px) {
+  body {
+    justify-content: center;
```

Comment:
- [ ] Firstly, I must commend your beautiful design on implementing the desktop version of the application. However, the requirement is to make it form a desktop view from a screen width of `768` or wider. I've attached a picture with the required screen size to show what your web app looks like.
- [ ] Screens are typically designed to be viewed at a specific resolution, with the most common resolution being `768 `pixels wide or wider for desktop screens. This is because most websites and web-based applications are optimized for this resolution, as it allows for the display of a wide range of content, including text, images, and multimedia. Additionally, having a wider screen allows for more content to be displayed on the screen at once, making it easier for users to navigate and interact with the website or application.

# Present design :
![image](https://user-images.githubusercontent.com/50990925/213591190-58700f60-1e9f-46c2-b46a-3c36f7828912.png)

# Expected Design:
![image](https://user-images.githubusercontent.com/50990925/213591328-2c9a83ef-aa52-4062-9898-d41b433fdd02.png)